src/plugins/waterfall-plugin.js

```diff
@@ -1,5 +1,5 @@
 import { deepMerge } from '../utils.js';
-import EventEmitter from 'events';
+import UIPlugin from './UI-plugin.js';
```

Comment:
File names are all in kebab case (lower case and minus signs). I would like to maintain consistency here as well.